tests/providers/amazon/aws/operators/test_eks.py

```diff
@@ -49,10 +52,16 @@
 NAME_LIST = ["foo", "bar", "baz", "qux"]
 
 
+class clusterParams(TypedDict):
+    cluster_name: str
+    cluster_role_arn: str
+    resources_vpc_config: Dict[Any, Any]
+
+
 class TestEksCreateClusterOperator(unittest.TestCase):
     def setUp(self) -> None:
         # Parameters which are needed to create a cluster.
-        self.create_cluster_params = dict(
+        self.create_cluster_params: clusterParams = dict(
```

Comment:
@potiuk please let me know your thoughts on this approach for fixing dict unpacking.

Reply:
Fixed all errors.